### PR TITLE
PAPI-2091: store metafield's open api spec

### PR DESCRIPTION
**Pull request by bc-tayvs**
# [PAPI-2091]

## What changed?
- Store metafield's open api spec

## Release notes draft

* 

## Anything else?

ping 

[PAPI-2091]: https://bigcommercecloud.atlassian.net/browse/PAPI-2091?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: reference/store_information.v2.yml [hunk 1]
```diff
@@ -98,9 +100,236 @@ paths:
...
+            The resource was not found.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/NotFound'
+              examples:
+                example-1:
+                  value:
+                    status: 404
+                    title: There was no order found with ID 1010
```
Comment (kskomro) [inline reference/store_information.v2.yml:317]: We don't check if related entities exits in metafields service, so example is a little inaccurate

**Review by kskomro — APPROVED** [review]

File: reference/store_information.v2.yml [hunk 2]
```diff
@@ -98,9 +100,236 @@ paths:
...
+      description: Gets a store metafield.
+      operationId: getStoreMetafield
+      responses:
+        '200':
+          description: |
+            A `Metafield` object.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MetafieldResponse'
```
Comment (6juara9) [inline reference/store_information.v2.yml:256]: It is a single API, we don't want to see a batch response here :) 

image

File: reference/store_information.v2.yml [hunk 3]
```diff
@@ -98,9 +100,236 @@ paths:
...
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '422':
+          description: |
+            The `Metafield` is not valid. This is the result of missing required fields or of invalid data. See the response for more details.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
```
Comment (6juara9) [inline reference/store_information.v2.yml:270]: Looks weird

image

Comment (bc-tayvs) [reply reference/store_information.v2.yml:270]: Examples are auto generated based on structure

**Review by bc-tgomez — APPROVED** [review]
Thanks for the wonderful job here! I'm going to pull it into a dev-docs branch and run some minor copy editing. We can get this published after we get the final sign-off.

**Pull request merged**